README.md

```diff
@@ -1 +1,32 @@
+
+**Winning code for other DrivenData competitions is available in the [competition-winners repository](https://github.com/drivendataorg/competition-winners).**
+
+## Winning Submissions
+
+Place | Team or User | Public Score | Private Score | Summary of Model
+--- | --- | ---   | ---   | ---
+1   | [vecxoz](https://www.drivendata.org/users/vecxoz/) | 6.677 | 6.256 | Create 3-channel images by concatenating an image frame with two prior timesteps. Use a TimeDistributed layer to process sequences of 24 images (8 3-channel images). Train 51 models with CNN-1D, CNN-3D, CNN-MEAN, and CNN-LSTM architectures using different backbones and different historical depths. Ensemble predictions using a weighted average optimized by pairwise greedy search over sorted predictions. Backbones include ResNet50, Inception, InceptionResNet, Xception, EfficientNet-B0, EfficientNet-B3, and VisionTransformer-L32. Features are 366 x 366 pixel images with flip and rotation augmentations and corresponding test time augmentations. Classification models using 156 separate classes also contribute to the final ensemble.
+2   | [KarimAmer](https://www.drivendata.org/users/KarimAmer/) | 6.867 | 6.417 | Fine-tune pre-trained CNNs, including ResNet50 and ResNet18, on sets of 3-channel images created by concatenating an image frame with two prior timesteps. Sets are composed of 1, 3, 6, 9, or 12 consecutive time steps, or 6 time steps using every other prior image (eg. images 1, 3, 5, 7, 9, 11). Apply test time augmentation to intermediate predictions. Features are 244 x 244 pixel images with flip, rotation, and time augmentations. Train an additional ~200 models with CNN-1D, Xgboost, LSTM, GRU, MLP, Transformer, Decision Tree, and Linear Regression architectures on intermediate predictions as well as 25-30 prior time steps using Deep Adaptive Input Normalization. Ensemble predictions using an ensemble selection algorithm with bagging.
+3   | [Daniel_FG](https://www.drivendata.org/users/Daniel_FG/) | 6.839 | 6.456 | Fine-tune 13 models on sequences of 8 or 12 3-channel images with CNN, RNN, GRU, and Transformer architectures, using different backbones and different historical depths. Fine-tune one model using Optical Flow Vectors from a sequence of 5 images spaced 1 hour apart. Backbones include VGG-11, VGG-16, ResNeXt101, SE_Resnet152, and SE_ResNeXt101. Multi-channel images are created from repeating a single channel or concatenating a monochrome image with two prior timesteps of varying depths using a block sampling strategy. Initial features are 244 x 244 pixel images with flip, rotation, and crop augmentations. Train a lightGBM on intermediate predictions, predictions from times t-1 through t-24, ocean, and relative time metadata to output final predictions.
+
+Additional solution details can be found in the `reports` folder inside the directory for each submission.
```

Comment:
LGTM! One thought: given the desire to use the results and the size of these ensembles, we might call out explicitly something like 
> Additional solution details, including practical tips for using subsets of the winning models, can be found in the `reports` folder inside the directory for each submission.

(and then confirming that we do that information in there where possible)